components/brave_wallet/browser/zcash/zcash_wallet_service.cc

```diff
@@ -267,7 +286,7 @@ absl::optional<std::string> ZCashWalletService::GetUnusedChangeAddress(
   // TODO(cypt4): this always returns first change address. Should return
   // first unused change address.
   return keyring_service_->GetZCashAddress(
-      account_id, mojom::ZCashKeyId(account_id.bitcoin_account_index, 1, 0));
+      account_id, mojom::ZCashKeyId(account_id.bitcoin_account_index, 0, 0));
```

Comment:
This now returns first receive address. Is this expected?

Reply:
Yes, it simplifies testing until discovery is implemented